samples/Net7Worker/EventHubCancellationToken.cs

```diff
@@ -17,6 +17,7 @@ public EventHubCancellationToken(ILoggerFactory loggerFactory)
 
         // Sample showing how to handle a cancellation token being received
         // In this example, the function invocation status will be "Cancelled"
+        <docsnippet_cancellation_token_throw>
```

Comment:
Don't we need to keep this behind a single line comment?

```suggestion 
        //<docsnippet_cancellation_token_throw>
```

Reply:
🤦‍♂️